README.md

```diff
@@ -26,6 +26,21 @@ or by downloading a single zip or phar file from our [latest release][latest-rel
   * [User Guide][docs-guide-v2]
   * [API Docs][docs-api-v2]
 
+### Getting Help
+
+Please use these community resources for getting help. We use the GitHub issues for tracking bugs and feature requests and have limited bandwidth to address them.
+
+* Ask a question on [StackOverflow](https://stackoverflow.com/) and tag it with `aws-php-sdk`
```

Comment:
Could you provide a cooler tag link like: http://stackoverflow.com/questions/tagged/aws-php-sdk ?